lib/graphiti/util/persistence.rb

```diff
@@ -205,17 +153,6 @@ def assign_temp_id(object, temp_id)
     object.instance_variable_set(:@_jsonapi_temp_id, temp_id)
   end
 
-  def iterate(only: [], except: [])
```

Comment:
this method is not removed, I had to make it public so I can access it from the adapter class. I can use `send` instead but I am not sure which choice is more clear. Suggestion?